### ISSUE TITLE
Support of Dynamic OPML import

### ISSUE THREAD

**Issue by tobiasholle** [spacecowboy/Feeder#392]
### Checklist

- [X] I have used the search function for [**OPEN**](https://github.com/spacecowboy/feeder/issues) issues to see if someone else has already submitted the same feature request.
- [X] I have **also** used the search function for [**CLOSED**](https://github.com/spacecowboy/feeder/issues?q=is%3Aissue+is%3Aclosed) issues to see if the feature was already implemented and is just waiting to be released, or if the feature was rejected.
- [X] I will describe the problem with as much detail as possible.
- [X] This request contains only one single feature, **not** a list of multiple (related) features.

### App version

2.6.33

### Where did you get the app from

Google Play

### Feature or improvement you want

Hi! 
It would be really great if Feeder would support Dynmaic OPML. I.e. FreshRSS offers Dynamic OPML interfaces and this would be very helpful for sharing feeds that automatically update. Are there any plans to add this option?

Best, Tobias

**Comment by spacecowboy** [comment]
I have no idea what Dynamic OPML interface means. Could you elaborate? Maybe post a screenshot if it's UI thing

**Comment by tobiasholle** [comment]
Sorry! You can find some information about it here, in application for FreshRSS: https://github.com/FreshRSS/FreshRSS/pull/4407

**Comment by spacecowboy** [comment]
TL;DR; 

Configure a tag to be populated from an OPML file uri (http://example.com/example.opml), updating for example every day.
